libs/packages/sam-formly/src/lib/formly/types/input.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+import { FieldType } from '@ngx-formly/core';
+
+@Component({
+  selector: 'sam-formly-field-input',

```

Comment:
We are also not using sam as the selector prefix

Reply:
updated to sds